### DISCUSSION TITLE
Connecting a interface to a rear-port with multiple position with multiple corresponding front ports

### DISCUSSION THREAD

**Discussion by romanmendelproject** [netbox-community/netbox#8150] [Q&A]
### NetBox version

v3.1.2

### Feature type

New functionality

### Proposed functionality

![image](https://user-images.githubusercontent.com/33754158/147100751-6af605af-8922-48c3-be70-d282fac59987.png)
Such a connection scheme is workable in the latest version of the netbox, however, tracing is not built, can you consider the possibility of introducing this functionality?

### Use case

Trace MPO interfaces using hydra cables

**Comment by jeremystretch** [comment]
Moved to discussions as no specific implementation has been proposed.
